torchdynamo/variables/builder.py

```diff
@@ -380,6 +382,10 @@ def index_source(key):
                 for k in ("start", "stop", "step")
             ]
             return SliceVariable(items, guards=make_guards(GuardBuilder.TYPE_MATCH))
+        elif isinstance(value, PyOperator):
+            return TorchPyOperator(
+                value, guards=self.make_guards(GuardBuilder.TYPE_MATCH)
```

Comment:
I wonder if TYPE_MATCH is enough.... is every PyOperator a different type?

Reply:
Good question - same type. I wonder if I should add a GuardBuilder.NAME_MATCH?